.github/workflows/build.yaml

```diff
@@ -58,3 +59,33 @@ jobs:
               exit 1
               ;;
           esac
+
+  build-windows:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: ["windows-latest"]
+        go: ["1.17.8"]
+    env:
+      GIT_ORG: ${{ github.repository_owner }}
+      GIT_REPO: ${{ github.events.repository.name }}
+      GO_VERSION: ${{ matrix.go }}
+    runs-on: ${{ matrix.os }}
+    steps:
+    # This step is required as long as the symlinks for client/v3/*_test.go -> tests/integration/clientv3/examples/*_test.go exist
+      - name: core.symlinks
+        run: |
+          git config --global core.symlinks true
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1      
+      - uses: WillAbides/setup-go-faster@v1.7.0
```

Comment:
Faster is not alwasy better. We should prefer official actions that are well maintained instead of trying a fork.
Please use normal `setup-go` and create separate issue/PR with proposal to migrate to setup-go-faster so we can discuss it independently.

Reply:
This should be actions/setup-go?